src/site/_data/i18n/courses.yml

```diff
@@ -3,3 +3,6 @@ learn_css:
     en: Learn CSS!
   description:
     en: "Let’s gleefully breakdown complex aspects of CSS into digestible episodes covering everything from accessibility to z-index."
+
+new:
+  en: new
```

Comment:
maybe we should put this in i18n.common since it seems like something we may use elsewhere on the site

Reply:
done